docs/user-guide/zonal-average.ipynb

```diff
@@ -10,6 +10,21 @@
     "This section demonstrates how to perform Zonal Averaging using UXarray, covering both non-conservative and conservative methods.\n"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    uxds["psi"].plot(cmap="inferno", periodic_elements="split", title="Global Field")
</pre><p> </p><p>This cell needs to be split after this <code>plot</code> line because  the resulting plot is getting bypassed with the remaining code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1403/discussion/#comment-2520082359'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>